src/test/java/org/xerial/snappy/SnappyTest.java

```diff
@@ -474,28 +481,35 @@ public void isValidArrayInputLengthForBitShuffleShuffle()
      */
     @Test(expected = SnappyError.class)
     public void isTooLargeDoubleArrayInputLengthForBitShuffleShuffle() throws Exception {
+        assumingCIIsFalse();
         BitShuffle.shuffle(new double[Integer.MAX_VALUE / 8 + 1]);
     }
 
     @Test(expected = SnappyError.class)
     public void isTooLargeFloatArrayInputLengthForBitShuffleShuffle() throws Exception {
+        assumingCIIsFalse();
         BitShuffle.shuffle(new float[Integer.MAX_VALUE / 4 + 1]);
     }
 
     @Test(expected = SnappyError.class)
     public void isTooLargeIntArrayInputLengthForBitShuffleShuffle() throws Exception {
+        assumingCIIsFalse();
         BitShuffle.shuffle(new float[Integer.MAX_VALUE / 4 + 1]);
     }
 
     @Test(expected = SnappyError.class)
     public void isTooLargeLongArrayInputLengthForBitShuffleShuffle() throws Exception {
+        assumingCIIsFalse();
         BitShuffle.shuffle(new long[Integer.MAX_VALUE / 8 + 1]);
     }
 
     @Test(expected = SnappyError.class)
     public void isTooLargeShortArrayInputLengthForBitShuffleShuffle() throws Exception {
+        assumingCIIsFalse();
         BitShuffle.shuffle(new short[Integer.MAX_VALUE / 2 + 1]);
+    }
 
-
+    private void assumingCIIsFalse() {
+        Assume.assumeTrue(System.getenv("CI").equals("false"));
```

Comment:
This part fails when CI env is not set:
```
[error] Caused by: java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because the return value of "java.lang.System.getenv(String)" is null
[error]     at org.xerial.snappy.SnappyTest.isTooLargeFloatArrayInputLengthForBitShuffleShuffle(SnappyTest.java:490)
[error]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[error]     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]     at java.lang.reflect.Method.invoke(Method.java:568)
```

Reply:
@aidanchiu1112 GitHub Actions will set CI=true (CI=false will never be set), so the condition should be fixed accordingly